rdm/cli.py

```diff
@@ -25,13 +25,36 @@ def cli(raw_arguments):
         init(args.output)
     elif args.command == 'pull':
         pull_requirements_and_reports(args.system_yml)
+    elif args.command == 'hooks':
+        install_hooks(args.dest)
 
 
 def init(output_directory):
     init_directory = pkg_resources.resource_filename(__name__, 'init')
     shutil.copytree(init_directory, output_directory)
 
 
+def install_hooks(dest=None):
+    hooks_source = pkg_resources.resource_filename(__name__, 'hooks')
+    if dest is None:
+        root = subprocess.check_output(['git', 'rev-parse', '--show-toplevel'])
```

Comment:
First of all: its great that you did not re-use the variable names here, and instead us `root` and `root_str`.

Second: In this situation, I tend to like making the longer-lived name (in this case `root`) take the nicer form.

For example, `root_as_bytes` and `root` instead of `root` and `root_str`.

I like this better because:

1. It indicates which variable is more important
2. It allows people reading the code to ignore the fact that `root` is a string, later on, since it is no longer relevant after the conversion.

Does this make sense?  please push back if you don't think it does

That said, both variable's scope is so short it doesn't really matter at the moment---but who knows how the code will evolve in the future.

Reply:
This makes sense; the change will be made in next push.